cmd/sovereignnode/config/common.go

```diff
@@ -15,3 +16,14 @@ func LoadSovereignGeneralConfig(filepath string) (*config.SovereignConfig, error
 
 	return cfg, nil
 }
+
+// LoadSovereignEpochConfig returns the epoch config necessary by sovereign by reading it from the provided file
+func LoadSovereignEpochConfig(filepath string) (*config.SovereignEpochConfig, error) {
```

Comment:
You don't need this. These flags are already in an existing file that is being loaded anyway.

Reply:
This is loading the SovereignEpochConfig struct